app/scripts/lib/auth-errors.js

```diff
@@ -279,6 +287,14 @@ define(function (require, exports, module) {
           return {
             param: err.param
           };
+        } else if (this.is(err, 'INVALID_RESUME_TOKEN_PROPERTY')) {
+          return {
+            property: err.property
```

Comment:
This was a premature request now that I see how things fit together. Keep as is.